### PR TITLE
dingo tutorial notebook

### LINKED ISSUE #14: Add Contribution guidelines
Add a detailed description of how other developers can contribute to `dingo`.

Example: https://github.com/GeomScale/volume_approximation/blob/develop/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hariszaf**
This is to provide the `dingo` library with an interactive tutorial: 

* it is available both as a Google Collab notebook and a Jupyter one
* it enables installation without the need of the `poetry` library 
* it includes examples of both FBA. FVA and flux sampling along with biological insight

### CODE REVIEW

**Comment by vissarion** [comment]
Great, thanks! 

Two questions only: 
- is the notebook synchronized with colab so when one change the one the other is also changed?
- should the notebook be placed in `doc` and not in the root of the project?

**Comment by hariszaf** [comment]
Hi @vissarion ! 

Thanks for reviewing the PR. 

Once you change the notebook, you need to do `File > Save a copy to GitHub` and select the branch you're working in.
Then, as the notebook has the same name git considers it as the same file and keeps its history. 

I have not strong opinion about moving the notebook under `doc`. I could do that if you d prefer so.

**Comment by vissarion** [comment]
> Once you change the notebook, you need to do File > Save a copy to GitHub and select the branch you're working in.
Then, as the notebook has the same name git considers it as the same file and keeps its history.

Great! So the procedure for future contributors of notebook will be (if I understand correctly)
- fork dingo
- edit Collab notebook
- save notebook changes  using `File > Save a copy to GitHub` to your fork
- open a PR for reviewing the changes

> I have not strong opinion about moving the notebook under doc. I could do that if you d prefer so.

What about putting it in a specific directory `tutorials` that also contains a README with information on how to run the notebook on Collab and locally as  well as a file CONTRIBUTING.md (as this)[https://github.com/GeomScale/volume_approximation/blob/develop/CONTRIBUTING.md] with the contributors instructions mentioned above?

@TolisChal what do you think?

**Comment by hariszaf** [comment]
If @TolisChal is ok with that i could go for it.

**Comment by hariszaf** [comment]
A directory called `tutorials` has beed added
The jupyter notebook and a `CONTRIBUTIND.md` file have beed moved there

**Comment by hariszaf** [comment]
Consider adding a `Cite` tab on the `README.md` of the repo

File: tutorials/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    python setup.py install --user 
+
+Once the last command is completed, you may check if everythings is fine by running some `dingo` tests
+
+    python tests/unit_tests.py
+
+
+If everything is ok, you will see something like this:
+
+[![asciicast](https://asciinema.org/a/3IwNykajlDGEndX2rUtc0D2Ag.svg)](https://asciinema.org/a/3IwNykajlDGEndX2rUtc0D2Ag)
```
Comment (vissarion) [inline tutorials/CONTRIBUTING.md:71]: this is very cool!!!
is it possible to embed a gif so the reader will not have to click to see the video

**Review by vissarion — APPROVED** [review]
I am OK with merging, thanks @hariszaf !

**Review by TolisChal — APPROVED** [review]

**Review by dymil — COMMENTED** [review]
It seems the wrong path was used for the last commit – this recreated the file at the repo root rather than under tutorials/

**Comment by vissarion** [comment]
Thanks for pointing this out! I agree, could you please open an issue (I think it is more clear than commenting in old PRs)? Or even better create a PR that fixes this issue directly.

**Comment by dymil** [comment]
Yeah, I just wanted to make sure @hariszaf got poked too, since there's that open PR with the community sampling and the notebook has some added cells for that which should probably be on that branch instead.

**Comment by vissarion** [comment]
OK, thanks! 
Please note that poking / referencing works also in all issues / discussions / PRs on github. The recommended method would be to open an issue by poking the user you want and refer to the current PR (like this https://github.com/GeomScale/dingo/pull/17) or open a new PR that fixes the current one with similar references. Thanks again for all your efforts!

**Pull request merged**